upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -824,6 +825,37 @@ func (tf *TemplateFunctions) GetNodeInstanceGroups() map[string]kops.InstanceGro
 	return nodegroups
 }
 
+type ClusterAutoscalerNodeGroup struct {
+	AutoScale *bool
+	MinSize   int32
+	MaxSize   int32
+	Other     string
```

Comment:
Can we call this `Name`?

Reply:
It comes from cluster-autoscaler args:
https://github.com/kubernetes/autoscaler/blob/f3242d8485ab3d53a6ac3fbe430918408295a8cf/cluster-autoscaler/main.go#L144-L146

Reply:
Ah okay. I guess it can be key/val and whatnot with some providers